src/ReleaseHistory.md

```diff
@@ -6,6 +6,7 @@
 * BUGFIX: Remove duplicated rule definitions when executing `match-results-forward` commands for results with sub-rule ids. [#2486](https://github.com/microsoft/sarif-sdk/pull/2486)
 * BUGFIX: Update `merge` command to properly produce runs by tool and version when passed the `--merge-runs` argument. [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
 * BUGFIX: Eliminate `IOException` and `DirectoryNotFoundException` exceptions thrown by `merge` command when splitting by rule (due to invalid file characters in rule ids).
+* BUGFIX: Fix NotYetAutoGenerated folder out of date issue by regenerate and manually sync the changes. [#2537](https://github.com/microsoft/sarif-sdk/pull/2537)
```

Comment:
>Fix NotYetAutoGenerated folder out of date issue by regenerate and manually sync the changes. [](http://example.com/codeflow?start=10&length=93)

How does this bug present from the customer's perspective? BUGFIX release history details should be of the format:
`BUGFIX: Fix <symptom from customer's perspective> [by <change details if appropriate>]...`

Reply:
right, the "NotYetAutoGenerated folder out of date" is not from customer's perspective.
how about:
BUGFIX: Fix classes inside NotYetAutoGenerated folder missing `virtual` keyword for public methods and properties, by regenerate and manually sync the changes.